Lib/pstats.py

```diff
@@ -500,7 +500,7 @@ def add_callers(target, source):
         if func in new_callers:
             if isinstance(caller, tuple):
                 # format used by cProfile
-                new_callers[func] = tuple(i[0] + i[1] for i in zip(caller, new_callers[func]))
+                new_callers[func] = tuple([i + j for i, j in zip(caller, new_callers[func])])
```

Comment:
Using *i* and *j* instead of indexing is an improvement.  Please drop the unnecessary square brackets though.